.github/workflows/health-issue-auto-closer.yml

```
@@ -1,8 +1,17 @@
+<<<<<<< HEAD
+name: Health Issue Auto-Closer (Legacy)
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 2: mapping values are not allowed in this context

(syntax-check)